webapp/app/dashboards/templates/timeseries_dashboard.html

```diff
@@ -208,7 +208,9 @@ <h5>Raw data</h5>
 <script src="{{ url_for('static', filename='javascript/utility.js') }}"></script>
 <script src="{{ url_for('static', filename='javascript/timeseries_dashboard.js') }}"></script>
 <!-- Chart.js -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.6.0/Chart.bundle.js"></script>
```

Comment:
oh, so it was previously getting from a CDN as well as having a local copy of everything?